translations.php

```diff
@@ -184,4 +184,8 @@
 
 // String added because original one was changed due to #3445024 issue.
 new TranslatableMarkup('Find people by name or email address');
-new TranslatableMarkup('You can enter or paste multiple entries separated by comma or semicolon');
\ No newline at end of file
+new TranslatableMarkup('You can enter or paste multiple entries separated by comma or semicolon');
+
+// String added because original one was changed due to #3452477 issue.
```

Comment:
@Macsym2018 Since we don't remove or change any string, but just add new we don't need to add changes in this file, so you can revert it.

Reply:
@ribel Changes were reverted.